cli.js

```diff
@@ -103,62 +131,43 @@ if (ciJobNumber() !== 1) {
 
 let getOptions
 if (argv['_'].length === 0) {
-  getOptions = readPkg().then(result => {
-    if (!result.pkg) {
-      throw ownError(
-        'Can not find `package.json`. ' +
-        'Be sure that you run Size Limit inside project dir.'
-      )
-    } else if (!result.pkg['size-limit'] && !result.pkg['sizeLimit']) {
-      throw ownError(
-        'Can not find `"size-limit"` section in `package.json`. ' +
-        'Add it according to Size Limit docs.' +
-        `\n${ EXAMPLE }\n`
-      )
-    }
-
-    if (result.pkg.sizeLimit) {
-      warn([
-        'Section name `"sizeLimit"` in package.json was deprecated.',
-        'Use `"size-limit"` for section name.'
-      ])
-    }
-
-    const limits = result.pkg['size-limit'] || result.pkg['sizeLimit']
-
+  getOptions = getConfig().then(limits => {
     if (configError(limits)) {
       throw ownError(
         configError(limits) + '. ' +
         'Fix it according to Size Limit docs.' +
         `\n${ EXAMPLE }\n`
       )
     }
-
-    return Promise.all(limits.map(limit => {
-      const cwd = path.dirname(result.path)
-      return globby(limit.path, { cwd }).then(files => {
-        if (files.length === 0) {
-          files = limit.path
-          if (typeof files === 'string') files = [files]
-        }
-        if (limit.babili) {
-          warn([
-            'Option `"babili": true` was deprecated.',
-            'Size Limit now supports ES2016 out of box.',
-            'You can remove this option.'
-          ])
-        }
-        return {
-          webpack: limit.webpack !== false,
-          bundle: result.pkg.name,
-          config: limit.config,
-          ignore: result.pkg.peerDependencies,
-          limit: limit.limit,
-          full: files.map(i => path.join(cwd, i)),
-          files
-        }
-      })
-    }))
+    return limits
+  }).then(limits => {
+    return readPkg().then(packageJson => {
+      return Promise.all(limits.map(limit => {
+        const cwd = path.dirname(packageJson.path)
```

Comment:
For `.size-limit` config you need to use its directory.

Reply:
Will fix it